### PR TITLE
Add additional alert freq options

### PR DESCRIPTION
**Pull request by SpennyNDaJets**
## Summary
When reviewing errors, we wanted to decrease the frequency we get alerted to 1 time/day for a specific error, but it was not an option.

Add additional frequencies to the Error Alert, and update the default frequency to 30 minutes

## How did you test this change?
1. Create a new error alert
- [ ] Values are displayed appropriately 
- [ ] Default value is changed to 1 hour

Screenshot 2023-06-28 at 3 24 25 PM

2. Create a new log alert
- [ ] Values are displayed appropriately 
- [ ] Default value is changed to 1 hour

Screenshot 2023-06-28 at 3 24 07 PM

## Are there any deployment considerations?
N/A

### CODE REVIEW

File: frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationConstants.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
+const SECONDS_PER_MINUTE = 60
+const MINUTES_PER_HOUR = 60
+const HOURS_PER_DAY = 24
+const DAYS_PER_WEEK = 7
+
+const SECONDS_PER_HOUR = SECONDS_PER_MINUTE * MINUTES_PER_HOUR
+const SECONDS_PER_DAY = SECONDS_PER_HOUR * HOURS_PER_DAY
+const SECONDS_PER_WEEK = SECONDS_PER_DAY * DAYS_PER_WEEK
+
+export const DEFAULT_FREQUENCY = `${SECONDS_PER_MINUTE * 30}` // 1800 seconds (30 minutes)
```
Comment (SpennyNDaJets) [inline frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationConstants.ts:10]: Updated from 15 seconds, but wanted to double check that this is an appropriate default value?

Comment (Vadman97) [reply frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationConstants.ts:10]: sounds good! we'll learn from our 1-day experiment if something larger is useful.

File: frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationCard.tsx [hunk 2]
```diff
@@ -930,8 +936,13 @@ export const AlertConfigurationCard = ({
...
 											text={
-												frequency === '1' ||
-												frequency === '60'
+												[
+													'1',
+													'60',
+													'3600',
+													'86400',
+													'604800',
+												].includes(frequency)
```
Comment (Vadman97) [inline frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationCard.tsx:945]: is there any way to reference the constants defined in `AlertConfigurationConstants.ts`?

**Review by Vadman97 — APPROVED** [review]
nice! and thanks for cleaning up the code by referencing a variable constant

**Review by Vadman97 — APPROVED** [review]

**Pull request merged**